packages/calypso-products/src/translations.tsx

```diff
@@ -261,6 +261,32 @@ export const getJetpackProductsTaglines = (): Record<
 	};
 };
 
+export const getJetpackProductDisclaimers = (): Record< string, TranslateResult > => {
+	const backupT1Disclaimer = translate(
+		'* Subject to your usage and storage limit. {{link}}Learn more{{/link}}.',
+		{
+			components: {
+				link: <a href="#backup-storage-limits-faq"></a>,
+			},
+		}
+	);
+	const backupT2Disclaimer = translate(
+		'* Subject to your usage and storage limit. {{link}}Learn more{{/link}}.',
+		{
+			components: {
+				link: <a href="#backup-storage-limits-faq"></a>,
+			},
+		}
+	);
+
```

Comment:
Felt it best to leave T1 and T2 separate in case we ever wanted to slightly change the text for one of them. I saw similar patterns elsewhere in this file